content/tutorials/jwt-authentication.textile

```diff
@@ -33,51 +33,54 @@ meta_keywords: "Ably realtime, JWT, authentication"
 
 When using token authentication, Ably recommends you instantiate the client library with a method to obtain tokens, rather than a token itself. This is due to the fact that tokens eventually expire, resulting in the connection being lost. By providing a method to obtain tokens instead, the Ably client library will automatically get a new token once the current one expires, maintaining a continuous connection.
 
 "JSON Web Tokens":https://jwt.io/ (JWTs) are an open, industry standard (RFC 7519) method for representing claims securely between two parties. JWT auth is especially helpful in scenarios where a single authentication scheme is to be used for all devices and browsers, and third-party platforms such as Ably. JWT auth is also a convenient way to authenticate your IoT devices or servers which are built using the platforms that Ably doesn't yet support. This is possible because you don't have to instantiate an Ably client when using the JWT auth scheme.
 
-Ably offers two different ways in which you can use a JWT to authenticate your users: 
+Ably offers two different ways in which you can use a JWT to authenticate your users:
 
-* You can use the JWT as the token itself by formatting it correctly (called an Ably JWT). 
-* You can pass to Ably a JWT of any structure that contains an Ably Token. 
+* You can use the JWT as the token itself by formatting it correctly (called an Ably JWT).
+* You can pass to Ably a JWT of any structure that contains an Ably Token.
 
 In this tutorial, the first of these options will be used.
 
-In the following steps you will: 
+In the following steps you will:
 
-* Build a simple login form that will authenticate a client with Ably, using a JWT. 
-* Learn how to use the "jsonwebtoken":https://www.npmjs.com/package/jsonwebtoken npm library, which is an implementation of JWT. This library allows your auth server to conveniently create a JWT by specifying only certain service specific data in the payload and headers. 
+* Build a simple login form that will authenticate a client with Ably, using a JWT.
+* Learn how to use the "jsonwebtoken":https://www.npmjs.com/package/jsonwebtoken npm library, which is an implementation of JWT. This library allows your auth server to conveniently create a JWT by specifying only certain service specific data in the payload and headers.
 * Create a web app that provides an endpoint that returns a JWT that can be used to autheticate your Ably client.
 
 So, let's get started and see a typical client-server architecture using JWT authentication.
 
 h2(#live-demo). Ably JWT auth example - Live Demo
 
-This section contains a live demo of the project you will create in this tutorial. 
+This section contains a live demo of the project you will create in this tutorial.
```

Comment:
This live demo doesn't really show anything useful - it just tells you that you are authenticated*. I think it would be much more valuable to display the JWT and the response here. WDYT?

* Also, I don't think we need the textbox output AND the alert box

Reply:
true, but on the tutorial page it does server a purpose, that is the user might not have a console open, so clicking login will have no "side-effect". On the localhost demo, i agree 100%. but I will leave it for now.

My preference would be to raise a response message in the UI.